cluster-autoscaler/simulator/drain_test.go

```diff
@@ -115,6 +115,7 @@ func TestGetPodsToMove(t *testing.T) {
 		Spec: apiv1.PodSpec{
 			Volumes: []apiv1.Volume{
 				{
+					Name: "empty-vol",
```

Comment:
Added volume name because it is a required field in Kubernetes. I realized the tests in this files were failing because I wasn't handling the case for empty volume name [here](https://github.com/kubernetes/autoscaler/pull/5594/commits/b0aca12b2685dfb9ffbbc98dda526f2099d9dec3#diff-a22fbdb27ac8bbd8f9c9597042678aa5f6541d207160adc775d56e13f3fe99beR259). This has been fixed in this PR [here](https://github.com/kubernetes/autoscaler/pull/5594/files#diff-a22fbdb27ac8bbd8f9c9597042678aa5f6541d207160adc775d56e13f3fe99beR271-R274).

I thought about removing the volume name above since I am handling the case for empty volume name but I kept it since this makes the manifest closer to actual manifest on Kubernetes. 

```
suraj@suraj:~/Downloads$ kubectl explain pod.spec.volumes | grep name
     Volume represents a named volume in a pod that may be accessed by any
   name	<string> -required-
     name of the volume. Must be a DNS_LABEL and unique within the pod. More
     https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
     PersistentVolumeClaim in the same namespace. More info:
```

Reply:
Ack, that makes sense.